Description: Remove cpp driver suffix.
 This disables behavior where using the cpp suffix for the name of the
 DPC++ compiler program results in running the compiler with driver
 mode CPP (preprocessor only). This will allow us to use the desired
 dpcpp suffix naming in the compiler sym links.
Author: Will French <will.french@canonical.com>
Last-Update: 2025-07-03
---
--- intel-dpcpp-6.1.0.orig/clang/lib/Driver/ToolChain.cpp
+++ intel-dpcpp-6.1.0/clang/lib/Driver/ToolChain.cpp
@@ -383,7 +383,6 @@ static const DriverSuffix *FindDriverSuf
       {"clang-gcc", nullptr},
       {"clang-cl", "--driver-mode=cl"},
       {"cc", nullptr},
-      {"cpp", "--driver-mode=cpp"},
       {"cl", "--driver-mode=cl"},
       {"++", "--driver-mode=g++"},
       {"flang", "--driver-mode=flang"},
